writer/writer/postgresql_backend/sql_database_backend_service.py

```diff
@@ -137,6 +142,13 @@ def insert_update_event(self, update_event, position) -> None:
         ]
         self.insert_db_event(update_event, arguments, position)
 
+    def insert_list_update_event(self, event: DbListUpdateEvent, position) -> None:
+        self.assert_exists(event.fqid)
+
+        model = self.read_database.build_model_ignore_deleted(event.fqid)
+        for translated_event in event.get_translated_events(model):
```

Comment:
this is the only time `get_translated_events` is called with a model. Please make an explicit function where the events gets translated to reduce the amount of side effects in `get_translated_events`.

Reply:
Moved it to a seperate method. Will have to adjust the tests